server/models/requestTypeModel.ts

```diff
@@ -16,6 +16,9 @@ interface RequestTypeInterface {
 type RequestTypeDocument = RequestTypeInterface & Document
 
 const RequestTypeSchema = new Schema({
+  requestGroup: {
```

Comment:
Could we get `requestGroup` added to `RequestTypeInterface`?

Reply:
Added